Cargo.toml

```diff
@@ -23,6 +23,7 @@ required-features = ["cli", "serde"]
 
 [dependencies]
 amplify = "3.13.0"
+bitcoin_scripts = "=0.8.2"
```

Comment:
why exact requirement here, not just "8.2"?

Reply:
dropped exact requirement